spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/BodyMethodGeneration.groovy

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2013-2019 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.contract.verifier.builder
+
+import org.apache.commons.text.StringEscapeUtils
+
+import org.springframework.cloud.contract.spec.internal.BodyMatcher
+import org.springframework.cloud.contract.spec.internal.MatchingType
+import org.springframework.util.SerializationUtils
+
+/**
+ * @author Marcin Grzejszczak
+ * @author Olga Maciaszek-Sharma
+ * @since 2.1.0
+ */
+trait BodyMethodGeneration {
```

Comment:
`@CompileStatic`?

Reply:
Some description of what it is used for?

Reply:
I've tried, but it causes issues with `JsonBodyVerificationBuilder` initialization.